privacyidea/lib/eventhandler/scripthandler.py

```diff
@@ -164,14 +186,23 @@ def do(self, action, options=None):
             proc_args.append("--logged_in_role")
             proc_args.append(logged_in_user.get("role", "none"))
 
+        rcode = 0
         try:
+            log.info("Starting script {script!r}.".format(script=script_name))
             p = subprocess.Popen(proc_args, cwd=self.script_directory)
-            log.info("Started script {script!r}:"
-                     " {process!r}".format(script=script_name, process=p))
+            if handler_options.get("background") == SCRIPT_WAIT:
+                rcode = p.wait()
+
         except Exception as e:
             log.warning("Failed to execute script {0!r}: {1!r}".format(
                 script_name, e))
             log.warning(traceback.format_exc())
```

Comment:
Should we raise an error here if ``raise_error`` is True?

Reply:
We did not raise an error before. I would like to keep it that way.
Raising an error could result in the request to fail. But imho the script is often just an addon, which should not interfer with the functinoality of e.g. the authentication.

Imagine an event handler, that is supposed to write a backup to a SMB mount on every 100th auth request. Now the SMB is down or someone change a password or deleted the script.
Full denial of service of your authentication ;-)

Reply:
But if I want to avoid that situation, I would probably set ``raise_error`` to False :-) If the script is important enough for me that I set ``raise_error=True``, I would probably also want the request to fail if the script is deleted and cannot be executed because of that.

Anyway, I'm also OK with keeping this as it is.

Reply:
You are adding Feature requests in the reviews! ;-)

Reply:
Currently "raise_error" can only be set for "background" mode.
Should we change this?

Reply:
Done in e2b1d60 :-)

Reply:
:+1: